api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2744,7 +2744,7 @@ public void saveOrder_shouldFailIfTheJavaTypeOfThePreviousOrderDoesNotMatch() th
 		Metadata metaData = new MetadataSources(standardRegistry).addAnnotatedClass(Allergy.class)
 				.addAnnotatedClass(Encounter.class).addAnnotatedClass(SomeTestOrder.class)
 				.addAnnotatedClass(Diagnosis.class).addAnnotatedClass(Condition.class)
-				.addAnnotatedClass(Visit.class).getMetadataBuilder().build();
+				.addAnnotatedClass(Visit.class).addAnnotatedClass(Drug.class).getMetadataBuilder().build();
```

Comment:
Can you format this line properly by placing `.addAnnotatedClass(Drug.class).getMetadataBuilder().build();` on a new line?

Reply:
Sure

Reply:
@alinmihaila99   i am also going to look through Drug.hbm.xml  and make some comparison here